src/js/components/HealthTab.js

```diff
@@ -131,7 +131,7 @@ class HealthTab extends React.Component {
       <div>
         <FilterHeadline
           currentLength={visibleData.length}
-          inverseStyle={false}
+          inverseStyle={true}
```

Comment:
I'm replying to this via email, since I can't see it on Github: That makes
sense, thanks for the explanation!

On Tue, May 3, 2016 at 11:58 PM, chris⚡ <notifications@github.com> wrote:

> In src/js/components/HealthTab.js
> <https://github.com/dcos/dcos-ui/pull/80#discussion_r61995731>:
>
> > @@ -131,7 +131,7 @@ class HealthTab extends React.Component {
> >        <div>
> >          <FilterHeadline
> >            currentLength={visibleData.length}
> > -          inverseStyle={false}
> > +          inverseStyle={true}
>
> I figured that the FilterHeadline is so specific that there is really no
> scenario we will be trying to apply other classes to it. Also the inverse
> styling is applied to it in a complicated, I think to expose all the styles
> is unnecessary (especially given that in this project we would never need
> to use them)
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dcos/dcos-ui/pull/80/files/2638efcc6cf3bdfa1ec88d61c8ab4a383abcd183#r61995731>
>